node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -284,8 +285,9 @@ class NodeRuntime private[node] (
             pm => HandleMessages.handle[Task](pm),
             blob => packetHandler.handlePacket(blob.sender, blob.packet)
           )
-      address = s"rnode://$id@$host?protocol=$port&discovery=$kademliaPort"
+      address = local.toAddress
       _       <- Log[Task].info(s"Listening for traffic on $address.")
+      _       <- EventLog[Task].publish(Event.NodeStarted(address))
```

Comment:
It just occurred to me: where are those events used? I think the PR introducing the EventLog should provide its first usage as well. Otherwise we might be very wrong about what's **atually** needed.